src/mavlink-router/ulog.cpp

```diff
@@ -55,6 +55,22 @@ bool ULog::_start_timeout()
     return true;
 }
 
+bool ULog::_stop_timeout()
+{
+    mavlink_message_t msg;
+    mavlink_command_long_t cmd;
+
+    bzero(&cmd, sizeof(cmd));
```

Comment:
Apparently [bzero](https://linux.die.net/man/3/bzero) is deprecated. Can you use ` mavlink_command_long_t cmd{};`?